### PR TITLE
Feat/assembly

### PR DESCRIPTION
**Pull request by jgfidelis**
- Display assembly options of assembly options of a cart item.
- Layout issues

### CODE REVIEW

File: react/components/ProductSummaryAttachmentList/AttachmentItem.js [hunk 1]
```diff
@@ -1,33 +1,57 @@
...
+const AttachmentItem = ({
+  productText,
+  price,
+  assemblyOptions,
+  showItemPrice,
+}) => {
+  const childrenAdded = (assemblyOptions && assemblyOptions.added) || []
+  const childrenRemoved = (assemblyOptions && assemblyOptions.removed) || []
+  const filteredChildrenAdded = reject(itemShouldHide, childrenAdded)
+  const fatherColor =
```
Comment (lbebber) [inline react/components/ProductSummaryAttachmentList/AttachmentItem.js]: :pick: 😬 
```suggestion
  const parentColor =
```

File: react/components/ProductSummaryAttachmentList/AttachmentItem.js [hunk 2]
```diff
@@ -1,33 +1,57 @@
...
-          sellingPriceLabelClass="dib"
-          sellingPriceClass="dib t-small c-muted-2"
-          showListPrice={false}
-          showLabels={false}
-          showInstallments={false}
-        />
-      )}
+    <div className={`${styles.attachmentItemContainer} flex flex-column pv1`}>
+      <div className={`flex items-center justify-between`}>
+        <span className={`t-small ${fatherColor} tl pr3`}>{productText}</span>
```
Comment (lbebber) [inline react/components/ProductSummaryAttachmentList/AttachmentItem.js]:
```suggestion
        <span className={`t-small ${parentColor} tl pr3`}>{productText}</span>
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Add capability to show assembly options of assembly options.
```
Comment (lbebber) [inline CHANGELOG.md]: :godeeper:

Comment (jgfidelis) [reply CHANGELOG.md]: :feelsgood:

**Pull request merged**